src/assets/custom/css/_sass/_services-curated-publications.scss

```diff
@@ -6,7 +6,7 @@
   }
 }
 
-.services__curated-publications--one-publication {
+.services__curated-publications--small-margin-top {
```

Comment:
Not sure about this rename? Want to make a case for it? Not a biggie if you want to merge but I think the classname should describe the what the element is not how it should be styled.

Reply:
Struggling to think of another name! - it's basically the curated publications if there's only one publication OR if there is no webinar. Any ideas?